.github/workflows/snyk_scan.yaml

```diff
@@ -0,0 +1,30 @@
+---
+name: Vanagon Snyk scan
+
+on: [pull_request]
```

Comment:
This should just be `on: merge` now given our recent conversations around `pull_request` and `pull_request_target`

Reply:
did you mean `on: push`?

Reply:
🤦 yes, sorry